### DISCUSSION TITLE
Scaling down ARGOCD_CONTROLLER_REPLICAS

### DISCUSSION THREAD

**Discussion by WojtekTomaszewski** [argoproj/argo-cd#12213] [Q&A] [answered]
Hi,

Is it safe to lower ARGOCD_CONTROLLER_REPLICAS value?  What will happen to clusters that lost its shard instance?

**Comment by jannfis** [comment, marked as answer]
It really depends. When you have pinned clusters to shards (e.g. by specifing `shard: X`, where `X` is the number of the shard), then clusters who are pinned to a shard may become unmanaged.

Otherwise, if you use automatic sharding, it should balance out automatically. Be sure to restart the statefulset of the application-controller, though, so the other pods will be notified of the change to the number of replicas.

**Comment by WojtekTomaszewski** [comment]
thanks!
